webpack/assets/javascripts/react_app/components/SearchButton/index.js

```diff
@@ -0,0 +1,6 @@
+import { connect } from 'react-redux';
+import SearchButton from './SearchButton';
+
+const mapStateToProps = ({ autocomplete }) => ({ searchQuery: autocomplete.searchQuery });
+
+export default connect(mapStateToProps)(SearchButton);
```

Comment:
why connect this small component to redux? couldn't we  pass the `searchQuery` from the parent?